tests/moe_ep/test_mxfp8_mxfp4_cutedsl_split_kernel.py

```
@@ -0,0 +1,443 @@
+    commutes with the gemm1 gate/linear interleave), and the gemm1→gemm2
+    hand-off round-tripped through MXFP8 as the kernel's epilogue does.
+    SwiGLU convention (defaults alpha=1, beta=0, no limit):
+    ``silu(gate) * linear`` with linear = first half of the gemm1 output.
+    """
+    _require_gpu_backend()
+
+    import torch
+
+    from flashinfer.moe_ep import EpLayout, SplitKernelContext
+
+    kernel, tw, fleet_params, w13, w2 = _make_backend_and_weights(
```

Ruff (0.16.1)

[warning] Unpacked variable `tw` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)